pages/linux/vnstati.md

```diff
@@ -1,6 +1,7 @@
 # vnstati
 
 > PNG image output support for vnStat.
+> More information: <https://linux.die.net/man/1/vnstati>.
```

Comment:
```suggestion
> More information: <https://manned.org/vnstati.1>.
```

We use manned.org when using man pages

Reply:
Never heard of manned.org before, but I updated it.